addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -253,6 +253,16 @@ class GVAR(actions) {
         consumeItem = 1;
         litter[] = {};
     };
+    class Grave: BodyBag {
+        displayName = CSTRING(DigGrave);
+        displayNameProgress = CSTRING(DiggingGrave);
+        icon = QPATHTOEF(medical_gui,ui\cross_grave.paa);
+        treatmentTime = QGVAR(treatmentTimeGrave);
+        condition = QFUNC(fnc_canDigGrave);
+        callbackSuccess = QFUNC(placeInGrave);
+        items[] = {"ACE_EntrenchingTool"};
```

Comment:
```suggestion
        items[] = {};
```

Reply:
The reason why I removed `ACE_EntrenchingTool` is because
1. the code suggested above takes care of that already
2. `ACE_EntrenchingTool` would be listed as a medical item in the arsenal